example/cumulus-tf/lambdas.tf

```diff
@@ -148,6 +148,7 @@ resource "aws_lambda_function" "ftpPopulateTestLambda" {
   role             = module.cumulus.lambda_processing_role_arn
   runtime          = "nodejs16.x"
   timeout          = 150
+  memory_size      = 512
```

Comment:
This mod should be included in the CL.   I realize it's not user facing, but it's project facing nonetheless.